Automattic/pocket-casts-ios
# Fix scroll indicator insets to adjust for mini-player

Fixes #1072 

This PR updates the code on multiple view controllers to adjust correctly the insets for the content and scroll indicators depending of the visibility of the mini-player.

| With Mini-Player | Without Mini-Player |
| - | - | 
| ![Simulator Screenshot - iPhone 15 Pro - 2024-05-17 at 10 47 47](https://github.com/Automattic/pocket-casts-ios/assets/651601/7b1a8ea9-3b03-449b-b59e-96d543547902) |  ![Simulator Screenshot - iPhone 15 Pro - 2024-05-17 at 10 47 30](https://github.com/Automattic/pocket-casts-ios/assets/651601/bb958503-011c-4b37-997f-bd55b220f30c) |

## To test

- Start the app
- On each of the main tabs check the scrolling limits with and without the mini-player visible and see if the content and scrollbar indicators are correct and adjust to the mini-player visibility
  - Podcasts check on the root screen and inside folders
  - Discovery: check on the root screen, and navigate inside the sections
  - Filters: check on the root screen and inside specific filters
  - Profile: check on the root screen and the multiple settings and account screens

## Checklist
